### PR TITLE
Toolbar and Headline Section

### PR DESCRIPTION
**Pull request by Zuwairanajma**
Here is the Toolbar and Headline Section for a mobile view portfolio project and I worked on the media query to ensure full width on all screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   </script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!-- Toolbar Header Section starts-->
+    <header> 
+        <div id= "head-wrap" class="headbar">
+    
+  <nav class = "headbar-menu">
+    <p class=""logo>My Logo</p>
```
Comment (TEMESGEN-G25) [inline index.html]: - [ ] I suggest using an anchor tag (`a`) for your logo, that'll navigate to the home page. That way your project will become more semantic & improve UI.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <h3 class="stext">I'm a Software Developer</h3>
+        <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+            <nav class="icon-wrap">
+                <ul class="navicon">
+               <li><a href="https://github.com/Zuwairanajma"><img src ="PImages/GitHubv1.png" alt = "Github Icon"></a></li>
+                <a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img src ="PImages/Linkedinv2.png" alt = "Linkedin Icon"></a></li> 
+                   <a href="https://twitter.com/@SadiqJuwairiyya"><img src ="PImages/Vector3.png" alt = "Angellist Icon"></a> </li>
+                   <a href="https://twitter.com/@SadiqJuwairiyya"><img src ="PImages/Twitter4.png" alt = "Twitter Icon"></a></li>
+                   <a href="https://twitter.com/@SadiqJuwairiyya"><img src ="PImages/Vector5.png" alt = "Medium Icon"></a></li>
```
Comment (TEMESGEN-G25) [inline index.html]: - [ ] Great job! On using the `ul` for your social icons, but all icons, except for the first one, seem to miss an opening `li` tag. I recommend adding them to not face more errors.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+<!-- Toolbar Header Section ends -->
+
+<main>
+    <!-- Headline Section Start -->
+    <section class="headline">
+    <div class="container_text">
+        <h1 class="ptext">Hey There. <br/>I'm Zuwairanajma.</h1>
+        <h3 class="stext">I'm a Software Developer</h3>
+        <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
```
Comment (TEMESGEN-G25) [inline index.html]: - [ ] I highly suggest that you follow the Figma template. Currently, there are some minor problems with your design:
<table>
<tr>
<th>Your design</th>
<th>Figma</th>
</tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

- Use the correct fonts for the title, sub-title & paragraph.
- Make sure to pack all of the elements to match the Figma design.

**Review by TEMESGEN-G25 — CHANGES_REQUESTED** [review]
## :yellow_circle: CHANGES REQUIRED :yellow_circle: 

## Hi @Zuwairanajma :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Used the correct elements to make the project semantic.
:heavy_check_mark: Almost matched the Figma template.



## Required Changes ♻️
_Check the comments under the review._
:x: I recommend you to include all main headers from the [README template](https://github.com/microverseinc/readme-template). Following Microverse's README template will help your project become more professional.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  font-style: normal;
+  width: 100%;
+  text-align: left;
+}
+
+.icon-wrap {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 151px 120px 24px;
```
Comment (franclobo) [inline styles.css]: - [ ] Kindly correct the padding and margin of the social icons to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Zuwairanajma, 

# Required Changes ♻️

soClose

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Best practices of HTML & CSS.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @Zuwairanajma 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Well structured files :white_check_mark:
  * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] All is correct keep doing the great work :clap: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**